### ISSUE TITLE
xsel or xclip?

### ISSUE THREAD

**Issue by Stockton-pr** [mawww/kakoune#1847]
|xsel --input --clipboard should copy any selection to the system clipboard, is that right?

I'm on hterm (secure shell Chromebook- emulates xterm AFAIK,  ssh to ubuntu 16 on aws).

I know that there are a few threads on this and stuff in wiki have read all and am stuck, any pointers very welcome!.

Also for linting do most folk just pipe % to a *-cli of their favourite linter or is there a better approach -am trying to get tidy working without success.

**Comment by Screwtapello** [comment]
xsel and xclip are specifically designed to interact with the X11 clipboard. I'm pretty sure the ChromeOS desktop is not X11-based, so you're very likely out of luck there.

You should be able to just select the text that appears on-screen and copy that, like in any other web-page. You may need to hold down shift (or some other modifier) to ensure your mouse-clicks are handled by Chrome, and not sent over SSH to Kakoune.

**Comment by jacobdufault** [comment]
I wonder if this can be used instead of xsel/xclip? https://github.com/chromium/hterm/blob/master/etc/osc52.sh

**Comment by Stockton-pr** [comment]
Thanks to both of you much appreciated and does confirm my intuition that this is going to be a nice experience.

I checked out the osc52 post it appeared as if by magic on the secure shell login page but tbh I came away none the wiser. Running kak in tmux appears to be the way forward and hence using tmux's functionality most of my use case for the clipboard is copying in and c-s v does that so no issues really and not a showstopper. Can someone close?
